kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kroxylicious/KroxyServiceResource.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.resources.kroxylicious;
+
+import java.util.concurrent.TimeUnit;
+
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServiceList;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.resources.ResourceOperation;
+import io.kroxylicious.systemtests.resources.ResourceType;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Kroxy service resource.
```

Comment:
We should avoid abbreviating Kroxylicious to Kroxy, both in classnames and comments.  Their name Kroxy existed already before we coined Kroxylicious.

Reply:
Agree. Thanks!